README.md

```diff
@@ -31,108 +64,107 @@ input:
         name:
           type:
             type_id: string
-steps:
+steps:  # The individual steps of the workflow
   example:
-    plugin: ghcr.io/janosdebugs/arcaflow-example-plugin
-    # step: step-id if the plugin has more than one step
-    # deploy:
-    #   type: docker|kubernetes
-    #   ... more options
+    plugin:
+      deployment_type: image
+      src: quay.io/arcalot/arcaflow-plugin-example
     input:
       name: !expr $.input.name
-output:
-  message: !expr $.steps.example.outputs.success.message
+outputs:  # The expected output schema and data for the workflow
+  success:
+    message: !expr $.steps.example.outputs.success.message
 ```
 
-As you can see, it has a `version`, `input`, a list of `steps`, and an `output` definition. Each of these keys is required in a workflow. These can be linked together using JSONPath expressions (not all features are supported). The expressions also determine the execution order of plugins.
+As you can see, a workflow has the root keys of `version`, `input`, `steps`, and
+`outputs`. Each of these keys is required in a workflow. Output values and inputs to
+steps can be specified using the Arcaflow
+[expression language](https://arcalot.io/arcaflow/workflows/expressions/). Input and
+output references create dependencies between the workflow steps which determine their
+execution order.
 
-You can now create an input YAML for this workflow: (save it to input.yaml)
+An input YAML file for this basic workflow may look like this:
 
 ```yaml
 name: Arca Lot
 ```
 
-If you have a local Docker / Moby setup installed, you can run it immediately:
+The Arcaflow engine uses a configuration to define the standard behaviors for deploying
+plugins within the workflow. The default configuration will use Podman as the container
+runtime and will set the log outputs to the `info` level.
```

Comment:
Technically, Podman can use any [OCI runtime](https://github.com/opencontainers/runtime-spec), but typically it uses either `runc` or `crun`.  I would finesse the issue here with verbiage like
>The default configuration will use Podman to run the container and will set the log outputs to the `info` level.